odoo/addons/test_new_api/models.py

```diff
@@ -60,10 +60,13 @@ def _inverse_display_name(self):
             # assign name of last category, and reassign display_name (to normalize it)
             cat.name = names[-1].strip()
 
-    def read(self, fields=None, load='_classic_read'):
+    def _read(self, fields):
+        # DLE P45: `test_31_prefetch`,
```

Comment:
What do those acronyms mean?

Reply:
Well, these comments were supposed to be deleted during the merge of 9920f20e4c7753bc17bea71dea3a90f7de687196. Except the comments that were still revelant to understand a specific line of code.

Basically, they were my comments during the development of the 13.0 ORM refactoring to help myself and other devs to tell for which reason I added, changed or deleted a line.

There were like 13x of those, so that was quite helpful to not get lost in the ORM refactoring abysses.

Reply:
Thank you for the explanation. I was afraid that it's another internet abbreviation I need to remember :smile:. Also, P45 looks like a reference to some specification like PEP8